portal/models/app_text.py

```diff
@@ -461,6 +461,7 @@ def subject(self):
                 self.error_msg = "Missing subject variable {}".format(e)
                 current_app.logger.error(self.error_msg +
                                          ": {}".format(self.url))
+                raise
         return self.error_msg
```

Comment:
Ah - great point.  Yes, I think the interface should be strict, so we can avoid unsightly messages landing in inboxes everywhere!